vm-gcp-go/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi-gcp/sdk/v6/go/gcp/compute"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+		// Import the program's configuration settings.
+		cfg := config.New(ctx, "")
+		machineType := "f1-micro"
```

Comment:
A more idiomatic way of doing it would be to use `config.Try`, then check for an error and set the value. See some of my Go templates for an example (or let me know if you need more detail).

Reply:
Thanks, will do!